R/chrome.R

```diff
@@ -6,19 +6,26 @@ Chrome <- R6Class("Chrome",
   public = list(
     #' @description Create a new Chrome object.
     #' @param path Location of chrome installation
+    #' @param use_srgb_color_profile Should a standard color profile (srgb) be
+    #'   used when taking screenshots? Forcing a standard profile increases
+    #'   portability of screenshots as colors will differ when the monitor of
+    #'   screenshotting machines have different color profiles. The downside is
+    #'   that colors of output may be slightly different than what you see in
+    #'   your browser.
+    #'   Equivalent to settings `args = '--force-color-profile="srgb"'`.
```

Comment:
Please re-wrap comments (Cmd-/ in RStudio) because they'll end up getting re-wrapped at some point in the future, and that will cause spurious changes.